### PR TITLE
Remove sources desnecessários do Gemfile

### LINKED ISSUE #46: Modificar declaração de múltiplos sources no Gemfile
Warning: this Gemfile contains multiple primary sources. Using `source` more than once without a block is a security risk, and may result in installing unexpected gems. To resolve this warning, use a block to indicate which gems should come from the secondary source. To upgrade this warning to an error, run `bundle config disable_multisource true`.

### PR DESCRIPTION
**Pull request by embs**
Bônus:
- Usa https para source rubygems

Closes #46

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,4 @@
-source 'http://rubygems.org'
-source 'http://gems.github.com'
```
Comment (embs) [inline Gemfile]: Ver https://github.com/OpenRedu/OpenRedu/commit/d29ee1c8c31ce9d4f740cd87f9a9a50d55b21f75#commitcomment-14560952

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,4 @@
...
-source 'http://reduadmin:pomp64bozos@the-shire.herokuapp.com/'
```
Comment (embs) [inline Gemfile]: Ver https://github.com/OpenRedu/OpenRedu/commit/1a401c929364b9c19989c9dd820129f0bbc3355e#commitcomment-14889762

**Comment by embs** [comment]
Tudo ok aqui, @OpenRedu/core?

**Comment by filipewl** [comment]
:+1:

**Pull request merged**